diff --git a/Package.swift b/Package.swift
index 94246dfd65d0f4e9d76182782798303b6af015f0..7d9d0078e6e29ac8326992bed1098b07c4119654 100644
--- a/Package.swift
+++ b/Package.swift
@@ -10,7 +10,7 @@ let package = Package(
             targets: ["SignInWithApple"])
     ],
     dependencies: [
-        .package(url: "https://github.com/ionic-team/capacitor-swift-pm.git", from: "7.0.0")
+        .package(url: "https://github.com/ionic-team/capacitor-swift-pm.git", from: "8.0.0")
     ],
     targets: [
         .target(
